### PR TITLE
[STREAM-146] - Update logging for new JSON-RPC commands.

### PR DESCRIPTION
**Pull request by zservies**

### CODE REVIEW

**Review by maxwellmooney13 — APPROVED** [review]

**Comment by zservies** [comment]
Dave didn't mention it in the ticket, but should we not also update messages for rejects/retracts - `sending jingle reject` to `sending jingle session-terminate (reject)` and `sending jingle session-terminate (retract)`?

File: src/webrtc.ts [hunk 1]
```diff
@@ -732,9 +734,9 @@ export class WebrtcExtension extends EventEmitter implements StreamingClientExte
     session.accepted = true;
 
     const details = this.getLogDetailsForPendingSessionId(sessionId);
-    this.logger.info('sending jingle proceed', details);
+    this.logger.info('sending jingle session-accept', details);
```
[the remark this answers is not in the document]
Comment (jensengar) [reply src/webrtc.ts:737]: You'll also need to add the corresponding log in `GenesysCloudMediaSession.accept()`

File: src/webrtc.ts [hunk 2]
```diff
@@ -106,10 +106,12 @@ export class WebrtcExtension extends EventEmitter implements StreamingClientExte
     );
 
     this.client.on('jingle:outgoing', (session: JingleSession) => {
+      this.logger.info('Emitting jingle media-session (session-init)', session);
```
[the remark this answers is not in the document]
Comment (jensengar) [reply src/webrtc.ts:109]: The sdp counterpart for this is in `webrtc.handleGenesysOffer()`

**Review by jensengar — CHANGES_REQUESTED** [review]

**Comment by jensengar** [comment]
> Dave didn't mention it in the ticket, but should we not also update messages for rejects/retracts - `sending jingle reject` to `sending jingle session-terminate (reject)` and `sending jingle session-terminate (retract)`?

terminate/session-terminate, yes.
reject and retract are stanza messages and aren't changing. we will continue to use them even when using sdpOverXmpp so  nothing needs to happen there.

File: src/types/genesys-cloud-media-session.ts [hunk 3]
```diff
@@ -324,6 +325,7 @@ export class GenesysCloudMediaSession {
       sessionId: this.id
     };
 
+    this.logger.info('sending jingle session-accept', params);
```
Comment (jensengar) [inline src/types/genesys-cloud-media-session.ts:328]: this should be `sending sdp answer`

**Review by jensengar — APPROVED** [review]

**Pull request merged**